composer.json

```diff
@@ -23,8 +23,10 @@
     "ext-gmp": "*",
     "lib-openssl": "*",
     "guzzlehttp/guzzle": "^6.2",
-    "web-token/jwt-signature": "^1.0",
-    "web-token/jwt-key-mgmt": "^1.0"
+    "web-token/jwt-signature": "^2.0",
+    "web-token/jwt-key-mgmt": "^2.0",
+    "web-token/jwt-signature-algorithm-ecdsa": "^2.0",
+    "web-token/jwt-util-ecc": "^2.0"
   },
   "require-dev": {
     "phpunit/phpunit": "^7.0",
```

Comment:
And it should probably be updated to phpunit 8.0 for 7.2/7.3 support - https://phpunit.de/getting-started/phpunit-8.html

Reply:
You are right, but I think that this should be done in a separate PR as it is not just a version bump

Reply:
Is it possible to avoid using "web-token/jwt-util-ecc"?